packages/create-nuxt3-app/src/index.ts

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+import {getUserInput} from "./prompts"
+import {generateWelcomeMessage} from './messages'
+// import {dirname, join} from 'path'
+import nodePlop from 'node-plop';
+
+(async () => {
+  generateWelcomeMessage();
+
+  const answers = await getUserInput()
+
+  /* Get the plop file from the external folder
+  const cnaTemplateDir = join(dirname(require.resolve('cn3a-plop-files/package.json')))
+  const plopFile = join(cnaTemplateDir, 'plopfile.js')
+   */
+  const plop = await nodePlop(`/Users/zoey/Documents/Development/SideStream/sidebase/create-nuxt3-app/packages/cn3a-plop-files/plopfile.js`);
```

Comment:
I have one remaining issue.

Currently I have this hardcoded to import the plopfile from my localfile system. Is there a way to import it from the package `cn3a-plop-files`?

From line 12-14 you can see my old approch (which I found on stackoverflow), however it does not seem to work anymore. 

Any tips would be appreaciated.